.gitignore

```diff
@@ -8,3 +8,4 @@ phpunit.xml
 .idea
 .phpstorm.meta.php
 .php_cs.cache
```

Comment:
Php cs cache can be left in, the other new lines should be removed.

Reply:
I add only html folder, because I use html coverage report for phpunit.

Reply:
Ok, but can you remove `.idea` and `.phpstorm.meta.php`?